### DISCUSSION TITLE
Custom Block plugin docs or referencs?

### DISCUSSION THREAD

**Discussion by 13316090972** [lowdefy/lowdefy#1549] [Questions and Help]
In New features of V4 ,Custom plugins make it easier and more standardized to implement custom blocks, connections, actions and operators. 
Is there any docs or reference for custom blocks' plugins?

**Comment by SamTolmay** [comment]
Hi @13316090972 

I am busy writing the documentation for plugins.

You can use this example repo as a reference: https://github.com/lowdefy/lowdefy-example-plugins
You can view the work-in-progress docs here: https://docs-git-docs-plugins-lowdefy.vercel.app
